src/admin/style.scss

```diff
@@ -130,6 +130,16 @@ h1 {
 /**
  * Wizards
  */
+// Only apply styles if there are sections and is immediate descendent.
+.newspack-wizard__content:has(> .newspack-wizard__sections) {
```

Comment:
Only apply styling to wizards wrapping components in `WizardsTab` component. See [example](https://github.com/Automattic/newspack-plugin/blob/feat/ia-audience/src/wizards/audience/views/configuration/settings.js#L164)